src/tf/tf.cpp

```diff
@@ -490,6 +492,25 @@ struct tf_parser
         return prog.add_instruction(op, {l0, new_weights});
     }
 
+    instruction_ref
+    parse_expanddims(const std::string&, const attribute_map&, std::vector<instruction_ref> args)
+    {
+        std::vector<size_t> input_dims = args[0]->get_shape().lens();
+        std::vector<int64_t> new_dims(input_dims.begin(), input_dims.end());
+        size_t num_dims = input_dims.size();
+        int32_t dim     = parse_axis(args[1]->eval().at<int32_t>(), num_dims);
```

Comment:
The `parse_axis` function shouldn't be needed if we set the `transpose` flag to false.

Reply:
fixed